Gofer.NET.Utils/TaskInfo.cs

```diff
@@ -43,7 +44,7 @@ public bool IsEquivalent(TaskInfo otherTaskInfo)
             }
 
             for (var i=0; i<Args.Length; ++i) {
-                if (!Args[i].Equals(otherTaskInfo.Args[i]))
+                if (!Equals(Args[i], otherTaskInfo.Args[i]))
```

Comment:
reason for this change?

Reply:
There will be a NulIRefException if Args[i] is null. Actually, I've catched one and did the change to mitigate it.